integration/single-node-slug-deploy/check.go

```diff
@@ -120,6 +134,26 @@ func generatePreparerPod(workdir string) (string, error) {
 	return manifestPath, err
 }
 
+func checkStatus(statusPort int, pod string) error {
```

Comment:
I also intend for this to be usable in https://github.com/anthonybishopric/p2/blob/balancer/integration/single-node-slug-deploy/check.go#L293 in the form of `checkStatus(43770, "hello")`.